app/src/main/java/com/example/unlibrary/exchange/ExchangeFragment.java

```diff
@@ -57,10 +60,23 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         for (Fragment f : getChildFragmentManager().getFragments()) {
             if (f instanceof BooksFragment) {
                 BooksFragment bookFragment = (BooksFragment) f;
+
                 bookFragment.setBooksSource(mViewModel);
+                bookFragment.setOnItemClickListener(this);
```

Comment:
Instead of making the entire fragment implement OnItemClickListener, we could pass in a lambda here to make it more consise + the body of the function is really short
```suggestion
                bookFragment.setOnItemClickListener(mViewModel::selectCurrentBook);
```

Reply:
+1 on this. Will be doing this in my own PR too.